instrumentation/activej-http-6.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/activejhttp/ActivejHttpServerAttributesGetter.java

```diff
@@ -21,6 +21,7 @@ final class ActivejHttpServerAttributesGetter
     implements HttpServerAttributesGetter<HttpRequest, HttpResponse> {
 
   @Override
+  @Nullable
```

Comment:
is this correct? `http.request.method` is marked as required in semconv

Reply:
you can add `@Nullable` to overrides that return `null`, but let's not add `@Nullable` to overrides that never return `null`

Reply:
done